views/admin/exhibits/exhibit-metadata-form.php

```diff
@@ -102,11 +113,45 @@
         </div>
     </section>
 </form>
-
+<div id="cover-image-panel" title="<?php echo html_escape(__('Choose a Cover Image')); ?>">
+    <div id="item-form">
+        <button type="button" id="revert-selected-item">Revert to Selected Item</button>
+        <button type="button" id="show-or-hide-search" class="show-form blue">
+            <span class="show-search-label"><?php echo __('Show Search Form'); ?></span>
+            <span class="hide-search-label"><?php echo __('Hide Search Form'); ?></span>
+        </button>
+        <a href="<?php echo url('exhibit-builder/items/browse'); ?>" id="view-all-items" class="green button"><?php echo __('View All Items'); ?></a>
+        <div id="page-search-form" class="container-twelve">
+        <?php
+            $action = url(array('module' => 'exhibit-builder',
+                'controller' => 'items', 'action' => 'browse'), 'default', array(), true);
+            echo items_search_form(array('id' => 'search'), $action);
+        ?>
+        </div>
+        <div id="item-select"></div>
+    </div>
+    <div id="cover-image-options">
+      <div class="options">
+        <button type="button" id="change-selected-item">Change Selected Item</button>
```

Comment:
Any visible text has to be passed through the `__()` function so we can translate it.